docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -284,6 +284,16 @@ web-0
 web-1
 ```
 
+Note, if you instead see 403 Forbidden responses for the above curl command,
+you will need to fix the permissions of the directory mounted by the `volumeMounts`
+(due to a [bug when using hostPath volumes](https://github.com/kubernetes/kubernetes/issues/2630)) with:
+
+```shell
+for i in 0 1; do kubectl exec web-$i -- sh -c 'chmod 755 /usr/share/nginx/html'; done
```

Comment:
You shouldn't need a shell to run `chmod`. It should work as just:

```shell
for i in 0 1; do kubectl exec web-$i -- chmod 755 /usr/share/nginx/html; done
```

Did you find that the permissions on `index.html` are fine?